### ISSUE TITLE
WiMAX: Send iRODS username

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-portal#1095]
For WiTEST (NYU), they want to help users set up irods. Currently, they create irods accounts. Once they are getting accounts from Orbit/Portal, they want to know if the user has an iRODS account already.

Send iRODS username in the LDIF to Orbit (format TBD). (Sync script must update this as needed.)

Must check to be sure no additional iRODS information needs to be sent.

 _Imported from trac ticket [#1095](http://trac.gpolab.bbn.com/proto-ch/ticket/1095),  created by ahelsing on 07-17-2014 at 16:10, last modified: 09-22-2014 at 11:26_

**Comment by ahelsing** [comment]
Capturing notes from email:
```
The portal stores the user's iRODS username when it creates the account. This is of the form geni-<portal username>. I can easily include this in LDIF and in OpenID if that's useful.

The portal does not know the user's iRODS PW.

The portal also creates iRODS groups. But I think for iinit purposes you don't care about that. (FWIW, the iRODS group names are of the form geni-<portal project name>.)

When creating accounts, the portal also feeds in the User's DN (which has a UID) in to iRODS (to theoretically support certificate based login to irods).

The portal also tries to guess things like irodsHost, irodsPort, irodsDefResources, irodsHome, irodsCwd, and irodsZone for filling in your irods env file. But this is usually something like emmy9.casa.umass.edu, 1247, demoResc, /geniRenci/home/<username>, /geniRenci/home/<username>, geniRenci respectively. I don't know if these are correct or not.

```

And:
```
To see a user's iRODS username when doing OpenID, they are available using OpenID Attribute Exchange (just as wimax/username is currently available)
• To receive iRODS username ask for http://geni.net/irods/username
• To receive iRODS zone ask for http://geni.net/irods/zone
```


_Trac comment by ahelsing on 07-17-2014 at 17:05_

**Comment by ahelsing** [comment]
fixed as #1167

_Trac comment by ahelsing on 09-22-2014 at 11:26_
